implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogMeterRegistry.java

```diff
@@ -230,6 +344,50 @@ private void addToMetadataList(Map<String, DatadogMetricMetadata> metadata, Mete
         }
     }
 
+    // VisibleForTesting
+    void writeMetricViaStatsd(Meter.Id id, @Nullable String suffix, double value, Statistic statistic,
+            @Nullable String overrideBaseUnit) {
+        if (statsDClient == null) {
+            return;
+        }
+
+        Meter.Id fullId = id;
+        if (suffix != null)
+            fullId = idWithSuffix(id, suffix);
+
+        Iterable<Tag> tags = getConventionTags(fullId);
+
+        List<String> tagsList = tags.iterator().hasNext() ? stream(tags.spliterator(), false)
+            .map(t -> "\"" + escapeJson(t.getKey()) + ":" + escapeJson(t.getValue()) + "\"")
+            .collect(toList()) : new ArrayList<String>();
+        if (config.hostTag() != null && !Objects.equals(config.hostTag(), "")) {
+            tagsList.add("host:" + config.hostTag());
```

Comment:
Pardon my ignorance, I'm no expert as fart as this `MeterRegistry` is concerned. Why are the previous tags wrapped in json (`"escapedJsonKey:escapedJsonValue"`) and `host` is not? Shouldn't it be `"host:..."`?

Reply:
host tag is special here as it can be set directly in the DatadogConfig. This seems to be because of the the previous direct-to-api method used prior. Normally dogstatsd agent will auto-collect the hostname(s) [cloud-environment aware] and auto-report this, but for backwards compatibility I included this specifically back in to make the migration easier if this config value is set.

Reply:
I see, thanks